model-optimizer/extensions/front/onnx/extract_image_patches_ext.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from extensions.ops.ExtractImagePatches import ExtractImagePatches
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.extractor import FrontExtractorOp
+from mo.front.onnx.extractors.utils import onnx_attr
+
+
+class ExtractImagePatchesExtractor(FrontExtractorOp):
+    op = 'ExtractImagePatches'
+    enabled = True
+
+    @classmethod
+    def extract(cls, node):
+        padding_mapping = {
+            b'SAME': 'same_upper', b'VALID': 'valid'
```

Comment:
nit: please split into two lines for the better look

Reply:
Done.